sc-server/src/utils/parser.hpp

```diff
@@ -0,0 +1,46 @@
+#include <iostream>
+#include <fstream>
+
+#include <boost/filesystem.hpp>
+#include <boost/config.hpp>
+#include <boost/program_options/detail/config_file.hpp>
+#include <boost/program_options/parsers.hpp>
+
+namespace pod = boost::program_options::detail;
+namespace fs = boost::filesystem;
+
+std::map<std::string, std::string> parse_config(std::string conf)
+{
+  fs::path conf_directory(conf);
+  conf_directory = conf_directory.parent_path();
+
+  std::ifstream config(conf);
+  if (!config)
+  {
+    std::cerr << "error" << std::endl;
```

Comment:
I think better to describe error

Reply:
Missed it, thanks. I'll fix it in the next commit.